ironfish/src/rpc/clients/socketClient.ts

```diff
@@ -34,10 +35,23 @@ export abstract class RpcSocketClient extends RpcClient {
   abstract client: IpcClient | net.Socket | null
   abstract isConnected: boolean
   abstract connection: Partial<RpcClientConnectionInfo>
+  authToken: string | null = null
+
+  constructor(logger: Logger, authToken?: string) {
+    super(logger)
+    if (authToken) {
+      this.authToken = authToken
+    }
```

Comment:
We usually use this syntax and don't initialize it in the class 

```suggestion
    this.authToken = authToken ?? null
```